Source/WebCore/rendering/EventRegion.cpp

```diff
@@ -213,8 +220,13 @@ bool EventRegionContext::shouldConsolidateInteractionRegion(RenderObject& render
 
         auto& ancestorBounds = discoveredIterator->value.first().rectInLayerCoordinates;
 
+        constexpr int loosenContainmentCheck = 3;
+        ancestorBounds.inflate(loosenContainmentCheck);
+        bool ancestorContainsElement = ancestorBounds.contains(bounds);
+        ancestorBounds.inflate(-loosenContainmentCheck);
```

Comment:
Let's copy the rect instead, it'll be clearer. Maybe `ancestorBoundsForLooseContainmentCheck`.